line_profiler/line_profiler.py

```diff
@@ -416,25 +416,32 @@ def load_stats(filename):
 
 
 def main():
-    def strictly_positive(option, opt, value, parser):
-        if value <= 0:
-            raise optparse.OptionValueError("option %s: floating-point value must be > 0, got %s" % (opt, value))
-        setattr(parser.values, option.dest, value)
-
-    usage = "usage: python -m line_profiler profile.lprof"
-
-    parser = optparse.OptionParser(usage=usage,
-                                   version=__version__)
-    parser.add_option('-u', '--unit', default='1e-6', type=float,
-        action='callback', callback=strictly_positive,
-        help="Output unit (in seconds) in which the timing info is to be displayed. Defaults to 1e-6.")
-    parser.add_option('-s', '--skip-zero', action='store_true', help="Hide functions which have not been called.")
-
-    options, args = parser.parse_args()
-    if len(args) != 1:
-        parser.error("Must provide a filename.")
-    lstats = load_stats(args[0])
-    show_text(lstats.timings, lstats.unit, output_unit=options.unit, stripzeros=options.skip_zero)
+    def positive_float(value):
+        val = float(value)
+        if val <= 0:
+            raise ArgumentError
+        return val
+
+    parser = ArgumentParser()
+    parser.add_argument('-V', '--version', action='version', version=__version__)
+    parser.add_argument(
+        '-u',
+        '--unit',
+        default='1e-6',
+        type=positive_float,
+        help="Output unit (in seconds) in which the timing info is displayed (default: 1e-6)",
+    )
+    parser.add_argument(
+        '-z',
```

Comment:
I do *like* -z, but I don't want to introduce a breaking change. It should be easy enough to just add "-s" as an accepted argument right? Although, it would be a bit annoying due to inconsistency with kernprof. 

Is there a way we can catch what the exact `sys.argv` string that triggered the arg parse option and give a deprecation warning if the user specifies `-s` rather than `-z`? (I think via a custom argparse.Action)?

Reply:
Hmm, that option was just added. It may be ok to just switch it to z and hope nobody depended on it in the small time the previous patch was released. Thoughts?

Reply:
Yes I think nobody is using `-s` except me as it was just added, I can also accept both options. Was the package even updated on pypi ?

Reply:
I did make a release on Nov 5th. I'm willing to be sneaky and simply change it and hope that nobody depended on it in the last month (they probably didn't). So, lets just make the change to `-z` and pretend like `-s` never happened.